script/DeployTimelockRolesUpgrader.s.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// slither-disable-start reentrancy-benign
+
+pragma solidity 0.8.26;
+
+import {Script} from "forge-std/Script.sol";
+import {TimelockRolesUpgrader} from "src/gov-action-contracts/TimelockRolesUpgrader.sol";
+import {SharedGovernorConstants} from "script/SharedGovernorConstants.sol";
+
+// Basic shared infrastructure for all deploy scripts
```

Comment:
@garyghayrat I'm not sure this is the right natspec or contract name. Perhaps we should inherit from BaseDeployer?

Reply:
https://github.com/ScopeLift/arbitrum-governor-upgrade/pull/11/commits/509970dbcd8bd3dbbda222f38b8721852a99d4b2